### ISSUE TITLE
[BUTTON IDEA]: Pay with buttons

### ISSUE THREAD

**Issue by RajMhatre20** [sButtons/sbuttons#1490]
**1. Name of button**: Pay with (icons)

**2. Description**: We can add a new category for Pay with / Continue with buttons.
 e.g. Pay with (Gpay icon), Pay with (Paypal icon).

**3. Button Type (Animated, Special, etc...)**: Payment Buttons

**4. Will you work on it?**: Yes, I want to work on it

**Comment by sid-khuntwal** [comment]
Can I work on this issue?

**Comment by shahednasser** [comment]
That's a good idea @RajMhatre20 . Yes, I think it's best to add it as a new button type. 

How about you start by creating the new type and adding one button of your choice in it. Try to use the Social buttons as a template on how to create a new button type.

Once that is done, @sid-khuntwal you can then add new buttons if you want under this new type.

**Comment by sid-khuntwal** [comment]
sure!

**Comment by shahednasser** [comment]
Hello @sid-khuntwal the payment button types has now been added. If you have an idea for a button to go into this category please submit a new issue with it and once approved and assigned you can work on it.

**Comment by RajMhatre20** [comment]
Hello @sid-khuntwal, I've created a new issue with button ideas. If you want you can work on it.

**Comment by sid-khuntwal** [comment]
I am on it  @RajMhatre20
